plugins/inventory/linode.py

```diff
@@ -66,6 +66,10 @@
 # Minimal example. `LINODE_ACCESS_TOKEN` is exposed in environment.
 plugin: community.general.linode
 
+# You can use Jinja to template the access token.
+plugin: community.general.linode
+access_token: "{{ lookup('ini', 'token section=your_username file=~/.config/linode-cli') }}"
```

Comment:
If the more modern syntax also works, please use it here:
```suggestion
access_token: "{{ lookup('ini', 'token', section='your_username', file='~/.config/linode-cli') }}"
```

Reply:
I think something like this would be good:
```suggestion
access_token: "{{ lookup('ini', 'token', section='your_username', file='~/.config/linode-cli') }}"
# For older Ansible versions, you need to write this as:
# access_token: "{{ lookup('ini', 'token section=your_username file=~/.config/linode-cli') }}"
```
(If you know from which version on it works, feel free to mention that as well :) )

Reply:
I like it. Committed. Thank you!